schema/google/showcase/v1beta1/showcase_v1beta1.yaml

```diff
@@ -78,8 +78,8 @@ http:
   - selector: google.longrunning.Operations.ListOperations
     get: '/v1beta1/operations'
   - selector: google.longrunning.Operations.GetOperation
-    get: '/v1beta1/{name=/operations/*}'
+    get: '/v1beta1/{name=operations/**}'
   - selector: google.longrunning.Operations.DeleteOperation
-    delete: '/v1beta1/{name=/operations/*}'
+    delete: '/v1beta1/{name=operations/**}:delete'
```

Comment:
Note to self: we shouldn't need the `:delete` suffix, this isn't a custom method.

Reply:
True. Thanks for the catch. (And fix)